FAQs/README.md

```diff
@@ -338,3 +341,50 @@ stages:
+
+> **Note:** If you do not have the permission to grant this permission, please contact your administrator to complete the setup using [this PowerShell script](../Scripts/ScriptToSetupAzureADApplicationForAzTSUI.ps1?raw=1). To run this script, you need to provide the subscription id and resource group name in which AzTS solution needs to be installed.
+
+
+**2. MS Graph "PrivilegedAccess.Read.AzureResources" and "Directory.Read.All" permissions for Scanner identity (MI)** 
+ 
+This permission is used to read data in your organization's directory, such as users, groups and apps and to validate Role-based access control (RBAC) using Azure AD Privileged Identity Management (PIM). 
+</br>If you do not have the required permission, please contact your administrator to get "PrivilegedAccess.Read.AzureResources" and "Directory.Read.All" permission for your scanner MI in Azure Active Directory using [this PowerShell script](../Scripts/ScriptToGrantGraphPermissionToScannerMI.ps1?raw=1). To run this script, you need to provide the object id of the user-assigned managed identity (scanner MI) which will be available in deployment log file.
+
+In case you are using Privileged Identity Management(PIM) for assigning access, please make sure that below settings are enabled:
+1. Go to Azure Portal.
+2. Navigate to AzTS host subscription -> AzTS host resource group.
+3. Go to AzTS Metadata Aggregator function app (AzSK-AzTS-MetadataAggregator-xxxxx).
+4. Go to Configurations under Settings.
+5. Check if setting AuthzSettings__IsPIMEnabled is available. If not, add new application setting 'AuthzSettings__IsPIMEnabled' with value as true. Click 'OK'. (This will enable AzTS to fetch role assignemnts using PIM API calls).
+6. Click 'Save'.
+7. Also check whether MS based graph access are granted to Scanner MI as mentioned above. If these permissions are granted post AzTS setup completion, then re-run AzTS setup installation command (Install-AzSKTenantSecuritySolution) with parameter -ScanIdentityHasGraphPermission:$true. More details can be found [here](../01-Setup%20and%20getting%20started/README.md#step-6-of-6-run-setup-command)
+
+From next AzTS scan run onwards, you should be able to see subsccriptions for which you have PIM based access.
+
+
+**3. MS Graph "User.Read.All" for AzTS Internal MI identity** 
+
+AzTS Internal MI is used by AzTS UI to read the list of security groups that the user is a member of. For this purpose, internal MI requires 'User.Read.All' permission. </br>If you do not have the required permission, please contact your administrator to get 'User.Read.All' permission for the internal MI in Azure Active Directory using [this PowerShell script](../Scripts/ScriptToGrantGraphPermissionToInternalMI.ps1?raw=1). To run this script, you need to provide the object id of the user-assigned managed identity (internal MI) which will available in deployment logs file.
```

Comment:
The AzTS Internal Managed Identity (MI) is utilized by the AzTS UI to retrieve the list of security groups that the user belongs to. In order to perform this task, the internal MI requires the 'User.Read.All' permission. If you don't currently have this permission, kindly reach out to your administrator to request the 'User.Read.All' permission for the internal MI in Azure Active Directory. Your administrator can grant this permission by utilizing a provided PowerShell script. To execute the script, you will need to provide the object ID of the user-assigned managed identity (internal MI), which can be found in the deployment logs file.

Reply:
Updated